tests/integration/framework/process/sentry/options.go

```diff
@@ -47,3 +48,9 @@ func WithCABundle(bundle ca.CABundle) Option {
 		o.bundle = bundle
 	}
 }
+
+func WithConfiguration(config *configv1alpha1.ConfigurationSpec) Option {
+	return func(o *options) {
+		o.configuration = config
+	}
+}
```

Comment:
We should always pass raw test strings for configuration files in integration tests. This ensures that we have proper test coverage and don't miss issues which arise from Marshal. See https://github.com/dapr/dapr/blob/8d7eaeea06541cdec356765dea1911bd95107b18/tests/integration/framework/process/daprd/options.go#L126

Reply:
Done